xml/templates/message_templates/participant_cancelled_html.tpl

```diff
@@ -34,8 +34,8 @@
      </tr>
      <tr>
       <td colspan="2" {$valueStyle}>
-       {$event.event_title}<br />
-       {$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"shortdate" == $event.event_start_date|crmDate:"shortdate"}{$event.event_end_date|crmDate:"Time"}{else}{$event.event_end_date|crmDate}{/if}{/if}
+       {event.title}<br />
+       {event.start_date|crmDate:"%A"} {event.start_date|crmDate}{if {event.end_date|boolean}}-{if '{event.end_date|crmDate:"%Y%m%d"}' === '{event.start_date|crmDate:"%Y%m%d"}'}{event.end_date|crmDate:"Time"}{else}{event.end_date|crmDate:"%A"} {event.end_date|crmDate}{/if}{/if}
```

Comment:
Trying to figure out if this makes sense. So this always hardcodes the weekday into the output? Any reason?

Reply:
@demeritcowboy I copied this from one of the main event templates - so from this template POV the reason is standardisation but I think the intent is not to show the date part twice if it starts & ends on the same day

Reply:
Ah, copy/paste used for good instead of evil.

Reply:
hah